.changeset/thick-birds-vanish.md

```diff
@@ -2,4 +2,5 @@
 "@fluidframework/tree": patch
 ---
 
-Fix document-corrupting bug when rebasing over move compositions
+Fix document-corrupting bug when rebasing over move compositions.
+Before this fix, if multiple users concurrently performed moves (possibly by reverting prior moves), there was a chance that the document would become corrupted.
```

Comment:
Nit. I think it's not strictly required but it helps to make the separation between summary and details clear
```suggestion
Fix document-corrupting bug when rebasing over move compositions.

Before this fix, if multiple users concurrently performed moves (possibly by reverting prior moves), there was a chance that the document would become corrupted.
```

Reply:
Thanks, change applied.